common/common-lang/src/test/java/com/twelvemonkeys/util/CollectionUtilTest.java

```diff
@@ -228,34 +228,12 @@ public void testArrayListIteratorRange() {
         assertCorrectListIterator(CollectionUtil.iterator(new String[] {"foo", "bar", "baz", "boo"}, 1, 2), new String[] {"bar", "baz"});
     }
 
-    @Test
```

Comment:
Why did you remove these tests? I don't see anything wrong with these?

Reply:
I could be wrong but it appears these tests don't test any of your code. The methods `assertCorrectListIterator` (which I made static just to be sure they didn't depend on the state of something set on the test class) have no dependencies on your custom collection classes and nothing passed into them in these test do either. It appears your testing `java.util.Arrays.asList`. 

Originally I thought all of your tests in this section did this but I see now you've created a test bed for you collection extentions. Did I miss something in this case?